src/hub/filters.py

```diff
@@ -17,26 +13,9 @@ def __init__(self, *args, **kwargs):
 
     def filter(self, qs, value):
         if value and any(v in ["score", "-score"] for v in value):
-            two_weeks_ago = timezone.now().date() - timedelta(days=14)
```

Comment:
instead of doing the filter here, can we remove this filter entirely? and change the `ordering=-paper_count / discussion_count`

Reply:
I think it would be

`?ordering=-paper_count,-discussion_count,-id`

Reply:
Does that even work? I'm trying that using `curl` but it's not working. It doesn't seem like Django supports complex ordering like that unless we have a custom Ordering Filter.